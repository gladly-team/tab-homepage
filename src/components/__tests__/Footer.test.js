/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Footer', () => {
  it('renders without error', () => {
    const Footer = require('../Footer').default
    shallow(<Footer />)
  })

  it('matches expected snapshot', function () {
    const Footer = require('../Footer').default
    const wrapper = shallow(<Footer />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
