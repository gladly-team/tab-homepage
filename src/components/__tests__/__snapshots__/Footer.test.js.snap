// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches expected snapshot 1`] = `
<div
  style={
    Object {
      "background": "rgba(128, 128, 128, 0.04)",
      "paddingBottom": 20,
      "paddingLeft": 40,
      "paddingRight": 40,
      "paddingTop": 1,
    }
  }
>
  <ForwardRef(Divider)
    style={
      Object {
        "marginBottom": 20,
        "width": "100%",
      }
    }
  />
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "flexWrap": "no-wrap",
      }
    }
  >
    <Link
      hoverStyle={Object {}}
      style={
        Object {
          "minWidth": 24,
        }
      }
      to="/"
    >
      <img
        src="test-file-stub"
        style={
          Object {
            "height": 43,
          }
        }
      />
    </Link>
    <div
      style={
        Object {
          "display": "flex",
          "flex": 1,
          "flexWrap": "wrap",
          "marginLeft": 30,
        }
      }
    >
      <div
        style={
          Object {
            "display": "flex",
            "flex": 1,
            "flexWrap": "wrap",
            "justifyContent": "flex-start",
          }
        }
      >
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="https://gladly.zendesk.com/hc/en-us/categories/201939608-Tab-for-a-Cause"
        >
          Help
        </Link>
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="/adblockers/"
        >
          Adblockers
        </Link>
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="/financials/"
        >
          Financials
        </Link>
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="/terms/"
        >
          Terms
        </Link>
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="/privacy/"
        >
          Privacy
        </Link>
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="/team/"
        >
          Team
        </Link>
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="/contact/"
        >
          Contact
        </Link>
        <Link
          hoverStyle={
            Object {
              "color": "#838383",
            }
          }
          style={
            Object {
              "color": "#cecece",
              "fontSize": 12,
              "margin": 20,
            }
          }
          to="/jobs/"
        >
          Jobs
        </Link>
      </div>
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "paddingTop": 4,
          }
        }
      >
        <Link
          hoverStyle={Object {}}
          style={
            Object {
              "margin": 12,
            }
          }
          to="https://www.tiktok.com/@tabforacause"
        >
          <IconWrapper
            hoverStyle={
              Object {
                "color": "#838383",
              }
            }
            style={
              Object {
                "color": "#cecece",
                "height": 20,
                "width": 20,
              }
            }
          >
            <ForwardRef(SvgIcon)
              style={
                Object {
                  "fill": "#fff",
                  "height": "38px",
                  "marginTop": "-3px",
                  "width": "38px",
                }
              }
              viewBox="0 0 50 50"
            >
              <path
                d="M41,4H9C6.243,4,4,6.243,4,9v32c0,2.757,2.243,5,5,5h32c2.757,0,5-2.243,5-5V9C46,6.243,43.757,4,41,4z M37.006,22.323 c-0.227,0.021-0.457,0.035-0.69,0.035c-2.623,0-4.928-1.349-6.269-3.388c0,5.349,0,11.435,0,11.537c0,4.709-3.818,8.527-8.527,8.527 s-8.527-3.818-8.527-8.527s3.818-8.527,8.527-8.527c0.178,0,0.352,0.016,0.527,0.027v4.202c-0.175-0.021-0.347-0.053-0.527-0.053 c-2.404,0-4.352,1.948-4.352,4.352s1.948,4.352,4.352,4.352s4.527-1.894,4.527-4.298c0-0.095,0.042-19.594,0.042-19.594h4.016 c0.378,3.591,3.277,6.425,6.901,6.685V22.323z"
              />
            </ForwardRef(SvgIcon)>
          </IconWrapper>
        </Link>
        <Link
          hoverStyle={Object {}}
          style={
            Object {
              "margin": 12,
            }
          }
          to="https://www.youtube.com/@tabforacause"
        >
          <IconWrapper
            hoverStyle={
              Object {
                "color": "#838383",
              }
            }
            style={
              Object {
                "color": "#cecece",
                "height": 20,
                "width": 20,
              }
            }
          >
            <Memo(ForwardRef(YoutubeIcon)) />
          </IconWrapper>
        </Link>
        <Link
          hoverStyle={Object {}}
          style={
            Object {
              "margin": 12,
            }
          }
          to="https://www.instagram.com/tabforacause/"
        >
          <IconWrapper
            hoverStyle={
              Object {
                "color": "#838383",
              }
            }
            style={
              Object {
                "color": "#cecece",
                "height": 20,
                "width": 20,
              }
            }
          >
            <Memo(ForwardRef(InstagramIcon)) />
          </IconWrapper>
        </Link>
        <Link
          hoverStyle={Object {}}
          style={
            Object {
              "margin": 12,
            }
          }
          to="https://www.facebook.com/TabForACause"
        >
          <IconWrapper
            hoverStyle={
              Object {
                "color": "#838383",
              }
            }
            style={
              Object {
                "color": "#cecece",
                "height": 20,
                "width": 20,
              }
            }
          >
            <Memo(ForwardRef(FacebookIcon)) />
          </IconWrapper>
        </Link>
        <Link
          hoverStyle={Object {}}
          style={
            Object {
              "margin": 12,
            }
          }
          to="https://twitter.com/TabForACause"
        >
          <IconWrapper
            hoverStyle={
              Object {
                "color": "#838383",
              }
            }
            style={
              Object {
                "color": "#cecece",
                "height": 20,
                "width": 20,
              }
            }
          >
            <Memo(ForwardRef(TwitterIcon)) />
          </IconWrapper>
        </Link>
      </div>
    </div>
  </div>
</div>
`;
